packages/platform-sdk-generator/src/model/Model.ts

```diff
@@ -94,6 +94,8 @@ export class Model {
         throw new Error("Method not implemented.");
       case "binary":
         return new BinaryType();
+      case "string":
```

Comment:
Maps can have a key of raw string type, we never ran into this before but the new stream apis added something with a Map<string, othertype> so had to add string type here